### ISSUE TITLE
Migrate to the desired migration.

### ISSUE THREAD

**Issue by Claud** [knex/knex#1550]
Please add this functionality. Like this:

`knex migrate:20160701152123_name`

**Comment by MatthewAry** [comment]
It's been over a year, I would like to see this too.

**Comment by olingern** [comment]
I think this can be closed as of #3205

**Comment by ivanovych666** [comment]
PR https://github.com/tgriesser/knex/pull/3416 now is merged and I think that this issue can be closed. Now we can migrate up/down the specified migration like this:
```
knex migrate:up 001_migration_name.js
knex migrate:down 001_migration_name.js
```
